js/defaults_dialog.js

```diff
@@ -393,6 +393,22 @@ helper.defaultsDialog = (function () {
                 key: "nav_wp_radius",
                 value: 5000
             },
+            {
+                key: "nav_fw_launch_max_angle",
+                value: 90
```

Comment:
I do think 90 degrees is a bit too steep. Most planes aren't overhead launched, so I think a 45-60 degree angle would be better for safety. Same applies to the other fixed wing defaults. In fact, you would never launch a plane with a tail overhead. So 45 degrees is plenty for them.

Reply:
You are right with the Plane with tail, that makes no sense. 
As a compromise for wings: Would you agree on a 75° limit for wings and 45° limit for planes with tail? 

for wings it makes also sense to keep it lower to avoid a power up if the plane is thrown in a vertical position where a gimbal lock can occur and you never know what the resulting direciton will be. 75° should be plenty for over head and also reduce the chance of a gimbal flip.

Reply:
Sounds good.